README.md

```diff
@@ -504,6 +504,37 @@ When this option is included in the command no other options are processed.
 nanoff --clearcache
 ```
 
+## Firmware archive
+
+By default, *nanoff* is designed to work with the latest version of the tool itself and of the firmware packages published in the online firmware repository. Command line arguments like
+
+```console
+nanoff --nanodevice --update --serialport COM9
+```
+
+will cause *nanoff* to check whether an update of the tool is available, download and deploy the latest version of the firmware for the connected device. This firmware has to match the NuGet packages used in the .NET **nanoFramework** applications, that may have to be updated to the latest version as well.
+
+For some projects auto-updating of tools, firmware and NuGet packages is not desirable, and a more controlled transition to new versions of the framework components is required. This can be done by downloading the firmware packages to a firmware archive location first, and then updating a nanoDevice's firmware from that archive. See the [documentation](https://docs.nanoframework.net/content/getting-started-guides/getting-started-versioning.html) for a more detailed description.
+
+Firmware can be archived via the *--updatefwarchive* option, e.g.:
```

Comment:
Better split these in sections with an appropriate  ### title (see the other sections).

Reply:
See next commit